File: src (VB.NET)/Z.ExtensionMethods.VB/Z.Core/System.Object[]/System.Type/Object[].GetTypeArray.vb

' Copyright (c) 2015 ZZZ Projects. All rights reserved
' Licensed under MIT License (MIT) (https://github.com/zzzprojects/Z.ExtensionMethods)
' Website: http://www.zzzprojects.com/
' Feedback / Feature Requests / Issues : http://zzzprojects.uservoice.com/forums/283927
' All ZZZ Projects products: Entity Framework Extensions / Bulk Operations / Extension Methods /Icon Library


Public Module Extensions_463

	''' <summary>
	'''     Gets the types of the objects in the specified array.
	''' </summary>
	''' <param name="args">An array of objects whose types to determine.</param>
	''' <returns>An array of  objects representing the types of the corresponding elements in .</returns>
	<System.Runtime.CompilerServices.Extension> _
	Public Function GetTypeArray(args As [Object]()) As Type()
		Return Type.GetTypeArray(args)
	End Function
End Module

File: src/Z.Core/System.Object[]/System.Type/Object[].GetTypeArray.cs
// Description: C# Extension Methods Library to enhances the .NET Framework by adding hundreds of new methods. It drastically increases developers productivity and code readability. Support C# and VB.NET
// Website & Documentation: https://github.com/zzzprojects/Z.ExtensionMethods
// Forum: https://github.com/zzzprojects/Z.ExtensionMethods/issues
// License: https://github.com/zzzprojects/Z.ExtensionMethods/blob/master/LICENSE
// More projects: http://www.zzzprojects.com/
// Copyright © ZZZ Projects Inc. 2014 - 2016. All rights reserved.
using System;

public static partial class Extensions
{
    /// <summary>
    ///     Gets the types of the objects in the specified array.
    /// </summary>
    /// <param name="args">An array of objects whose types to determine.</param>
    /// <returns>An array of  objects representing the types of the corresponding elements in .</returns>
    public static Type[] GetTypeArray(this Object[] args)
    {
        return Type.GetTypeArray(args);
    }
}